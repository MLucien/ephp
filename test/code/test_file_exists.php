<?php

if (!file_exists("this_file_doesnt_exist")) {
    print "perfect! file doesn't exist\n";
}

if (file_exists(__FILE__)) {
    print "yes! this file " . __FILE__ . " exists!\n";
}
