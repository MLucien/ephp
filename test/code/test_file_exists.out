perfect! file doesn't exist
yes! this file {{CWD}}/test/code/test_file_exists.php exists!
